### PR TITLE
Feature/add disposal fields support

### PR DESCRIPTION
**Pull request by dkumanov**
Expected Approvals: 2

### CODE REVIEW

File: src/main/java/com/documaster/rms/noark/ws/constants/Kassasjonshjemmel.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.documaster.rms.noark.ws.constants;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Kassasjonshjemmel extends NoarkEnum {
```
Comment (teodoradobreva) [inline src/main/java/com/documaster/rms/noark/ws/constants/Kassasjonshjemmel.java:6]: Should this list value and the other new one be added to `NoarkObjectMappings#createCodeValueMappings`?

**Review by teodoradobreva — CHANGES_REQUESTED** [review]
No other comments.

File: src/main/java/com/documaster/rms/noark/ws/constants/Kassasjonshjemmel.java (hunk 1 above)
Comment (dkumanov) [reply src/main/java/com/documaster/rms/noark/ws/constants/Kassasjonshjemmel.java:6]: Added with https://github.com/documaster/ws-client-n5-java/pull/27/commits/84b7aaaa9dac2d892c8b473a1c377dc9ef5ee6ab

**Review by teodoradobreva — APPROVED** [review]

**Review by ivaylomitrev — APPROVED** [review]

**Comment by dkumanov** [comment]
Squashed the 4th commit into the 1st one. No other changes. Preparing for merge.

**Pull request merged**